gradle.properties

```diff
@@ -18,9 +18,12 @@
 buildKotlinVersion=1.3.50-dev-44
 buildKotlinCompilerRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_dev_Compiler),number:1.3.50-dev-44,pinned:true/artifacts/content/maven
 remoteRoot=konan_tests
-kotlinCompilerRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_dev_Compiler),number:1.3.50-dev-259,branch:default:true,pinned:true/artifacts/content/maven
-kotlinVersion=1.3.50-dev-259
-testKotlinVersion=1.3.50-dev-259
+kotlinCompilerRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_dev_Compiler),number:1.3.50-dev-267,branch:default:true,pinned:true/artifacts/content/maven
+kotlinVersion=1.3.50-dev-267
+kotlinStdlibRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_dev_Compiler),number:1.3.50-dev-280,branch:default:true,pinned:false/artifacts/content/maven
+kotlinStdlibVersion=1.3.50-dev-280
+testKotlinCompilerVersion=1.3.50-dev-267
+testKotlinStdlibVersion=1.3.50-dev-280
```

Comment:
I believe after this change we'll always be able to use the same version of tests as the version of common stdlib, so we can join `kotlinStdlibVersion` and `testKotlinStdlibVersoin`. But if you have considerations why it still might be needed, then keep it separate.